### ISSUE TITLE
BAvatar renders alt text when an image fails to load

### ISSUE THREAD

**Issue by DutchGerman** [bootstrap-vue-next/bootstrap-vue-next#2280]
### Describe the bug

While using `BAvatar` with a dead link in the src prop, the alt text is rendered instead of the `text` prop or the default avatar image.

![image](https://github.com/user-attachments/assets/58686527-a0e4-46c0-b0eb-e32cd316e54b)

### Example 1

```js
<b-avatar src="https://i-do-not-exists-foobar.com/foobar.png" text="fallback text" />
```

**Expectation:** "fallback text" is rendered
**Reality:** The text `avatar` (the default `alt` prop) is rendered.

### Example 2

```js
<b-avatar src="https://i-do-not-exists-foobar.com/foobar.png"  />
```

**Expectation:** the default avatar image (`person-fill` icon) is rendered
**Reality:** The text `avatar` (the default `alt` prop) is rendered.

---
### Documentation:

In the documentation, the behavior is described as followed.

> If the image fails to load, the avatar will fall back to the value of the text prop. If the text props is not provided, then the default avatar image will be shown. Also, when the image fails to load, the img-error event will be emitted
https://bootstrap-vue-next.github.io/bootstrap-vue-next/docs/components/avatar.html

### Reproduction

https://stackblitz.com/edit/github-uewrop?file=src%2Fcomponents%2FComp.vue

### Used Package Manager

yarn

### Validations

- [X] Have tested with the latest version. This is still alpha version and sometime things change rapidly.
- [X] Follow our [Code of Conduct](https://github.com/bootstrap-vue-next/bootstrap-vue-next/blob/main/CODE_OF_CONDUCT.md)
- [X] Read the [Contributing Guide](https://github.com/bootstrap-vue-next/bootstrap-vue-next/blob/main/CONTRIBUTING.md).
- [X] Check that there isn't already an issue that reports the same bug to avoid creating a duplicate.
- [X] Check that this is a concrete bug. For Q&A, please open a GitHub Discussion instead.
- [X] The provided reproduction is a [minimal reproducible](https://stackoverflow.com/help/minimal-reproducible-example) of the bug.
